reference/store_logs.v3.yml

```diff
@@ -0,0 +1,240 @@
+openapi: 3.0.0
+x-stoplight:
+  id: 984571274919c
+info:
+  title: Store Logs V3 API
+  version: ''
+  description: Get store logs
+servers:
+  - url: 'https://api.bigcommerce.com'
+paths:
+  '/stores/{store_hash}/v3/store/systemlogs':
+    get:
+      summary: Get System Logs
+      description: |- 
+        Get system logs 
+
+        <!-- theme:info -->
+        > #### Note
+        > The documented values for the `module` query parameter are only some of the possible values. Other modules include payment providers and carrier shipping methods. Use this endpoint without query parameters to see all system log entries from the store and their module values.   
+
+      tags:
+        - System Logs
+      operationId: get-sites
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/SystemLog'
+                  meta:
+                    $ref: '#/components/schemas/IndexMeta'
+      parameters:
+        - schema:
+            type: string
+          in: path
+          required: true
+          name: store_hash
+        - schema:
+            type: integer
+            default: 50
+          in: query
+          name: limit
+          description: Results per page
+        - schema:
+            type: integer
+            default: 1
+          in: query
+          name: page
+          description: Numeric page of results
+        - schema:
+            type: string
+          in: query
+          name: 'date_created:min'
+          description: Minimum date
+        - schema:
+            type: string
+          in: query
+          name: 'date_created:max'
+          description: Maximum date
+        - schema:
+            type: string
+            enum:
+              - general messages
+              - payment gateways
+              - shipping quotes
+              - tax quotes
+              - promotion
+              - notification system
+              - email integration
+              - ebay
+              - shopping comparison
+              - order settings
+              - square
+              - facebook
+              - pinterest
+              - design
+              - inventory
```

Comment:
@bookernath Where can I find a list of all the modules? The ones I listed here are from my sandbox store, but was hoping to tell developers all the other types that they can query for. I found some on the UI (screenshot), though I realized the UI display term didn't always match the query term that developers can use.
![Type](https://user-images.githubusercontent.com/96258747/175103152-d05b5e9f-b92d-4cf2-b45b-2fd3dc11fa1c.png)